tests/s-exp-mixed.c

```diff
@@ -1,3 +1,40 @@
+#ifdef FASTCALL
+__attribute__((fastcall)) 
```

Comment:
I think you'd better making a new test case for the fastcall and leave this as is.

Reply:
it is need for test 085 about argument register. not fastcall.
without fastcall, x86 doesn't use register for passing argument.

Reply:
In the end we need to support both fastcall and normal calls seamlessly.  Just changing the test code to use fastcall only not to fail is meaningless.  Please let this test fail and add a new test case for fastcall (ignoring other archs for now).